README.md

```diff
@@ -40,17 +40,5 @@ See a code sample to manage trusted signing certificates and keys for Isolated p
 
 - [EGo's code sample using Azure Attestation](https://github.com/edgelesssys/ego/tree/master/samples/azure_attestation)
 - [Azure Attestation integration in SCONE platform](https://github.com/scontain/scone-azure-integration)
+- [Sample code for doing Microsoft Azure Attestation in Occlum](https://github.com/occlum/occlum/tree/master/demos/remote_attestation/azure_attestation)
```

Comment:
_"doing"_ - this is a vague word.  What specifically is performed?  Would a more precise description be "Sample code to generate and verify an SGX quote using Microsoft Azure Attestation in Occlum"?  Or perhaps "Azure Attestation integration in Occlum"?

Reply:
updated